test/fromPairsMapNonNil.spec.js

```diff
@@ -0,0 +1,36 @@
+const fromPairsMapNonNil = require('../src/fromPairsMapNonNil')
+const chai = require('chai')
+const sinon = require('sinon')
+const { expect } = chai
+chai.use(require('sinon-chai'))
+
+describe('fromPairsMapNonNil', function () {
+  it('should return an object', function () {
+    const array = ['a', 'b', null]
+    const expectedResult = {
+      a: 'a',
+      b: 'b',
+    }
+    const result = fromPairsMapNonNil(array, (v) => {
+      return !!v ? [v, v] : null
```

Comment:
You probably don't need the `!!v` here

```suggestion
      return v ? [v, v] : null
```